### PR TITLE
Entities

### PR DESCRIPTION
**Pull request by IsaSca**
entities done

### CODE REVIEW

File: src/main/java/me/jcsawyer/classroombot/entities/Session.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import java.time.OffsetDateTime;
+import java.util.List;
+
+public class Session {
+
+    private OffsetDateTime dateStarted;
+    private OffsetDateTime dateEnded;
+    private final long textChannelId;
+    private final long voiceChannelId;
+    private final long refisterId;
```
Comment (jcsawyer123) [inline src/main/java/me/jcsawyer/classroombot/entities/Session.java]: Typo

File: src/main/java/me/jcsawyer/classroombot/entities/Session.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    private List<Course> courses;
+    private List<Student> students;
+
+
+    public Session(OffsetDateTime dateStarted, OffsetDateTime dateEnded, long textChannelId, long voiceChannelId, long refisterId, long roleId, int maxAttendance, List<Course> courses, List<Student> students) {
+        this.dateStarted = dateStarted;
+        this.dateEnded = dateEnded;
+        this.textChannelId = textChannelId;
+        this.voiceChannelId = voiceChannelId;
+        this.refisterId = refisterId;
```
Comment (jcsawyer123) [inline src/main/java/me/jcsawyer/classroombot/entities/Session.java]: Typo

File: src/main/java/me/jcsawyer/classroombot/entities/Session.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        this.maxAttendance = maxAttendance;
+        this.courses = courses;
+        this.students = students;
+    }
+
+    public Session(OffsetDateTime dateStarted, long textChannelId, long voiceChannelId, long refisterId, long roleId, int maxAttendance, List<Course> courses, List<Student> students) {
+        this.dateStarted = dateStarted;
+        this.textChannelId = textChannelId;
+        this.voiceChannelId = voiceChannelId;
+        this.refisterId = refisterId;
```
Comment (jcsawyer123) [inline src/main/java/me/jcsawyer/classroombot/entities/Session.java]: Typo

File: src/main/java/me/jcsawyer/classroombot/entities/Session.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+    public long getTextChannelId() {
+        return textChannelId;
+    }
+
+    public long getVoiceChannelId() {
+        return voiceChannelId;
+    }
+
+    public long getRefisterId() {
```
Comment (jcsawyer123) [inline src/main/java/me/jcsawyer/classroombot/entities/Session.java]: Typo

**Review by jcsawyer123 — CHANGES_REQUESTED** [review]
Stop making typos

**Review by hexillium1 — APPROVED** [review]

**Pull request merged**